pkg/provisioner/ironic/testserver/server.go

```diff
@@ -68,6 +84,16 @@ func (m *MockServer) Endpoint() string {
 	return response
 }
 
+//Client returns a client
+func (m *MockServer) Client() *http.Client {
+	return m.Client()
```

Comment:
Sorry a typo here, it's instead `return m.server.Client()`

Reply:
Done